consensus/misc/eip6110.go

```diff
@@ -29,8 +29,11 @@ const (
 	BLSPubKeyLen             = 48
 	WithdrawalCredentialsLen = 32 // withdrawalCredentials size
 	BLSSigLen                = 96 // signature size
+
 )
 
+var depositTopic = libcommon.HexToHash("0x649bbc62d0e31342afea4e5cd82d4049e7e1ee912fc0889aa790803be39038c5")
```

Comment:
We need to double check whether this one is OK for Gnosis/Chiado

Reply:
OK, seems to be fine for Gnosis as well: https://gnosisscan.io/address/0x0B98057eA310F4d31F2a452B414647007d1645d9#events